amqp.sln

```diff
@@ -132,15 +132,15 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Amqp.NetMF44", "netmf\Amqp.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Test.Amqp.NetMF", "netmf\Test.Amqp.NetMF.csproj", "{2C4CBC7A-16BD-4F27-86BA-3CF38BE0F426}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Amqp.Net", "csproj\Amqp.Net.csproj", "{92153715-1D99-43B1-B291-470CF91A156D}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Amqp.Net", .csproj\Amqp.Net.csproj", "{92153715-1D99-43B1-B291-470CF91A156D}"
```

Comment:
I guess the changes in this file are done by some tool and not intentional. They make the sln file format invalid.

Reply:
They are indeed made by a tool and they are unintentional. Working on fixing this.